web/src/dropdown_widget.ts

```diff
@@ -106,6 +106,7 @@ export class DropdownWidget {
     disable_for_spectators: boolean;
     dropdown_input_visible_selector: string;
     prefer_top_start_placement: boolean;
+    item_clicked = false;
```

Comment:
This is meant to be a fully general module, so added a comment noting this should either be made part of the core system or otherwise reworked.

``` diff
diff --git a/web/src/dropdown_widget.ts b/web/src/dropdown_widget.ts
index d3ff771e08..b1861b64f9 100644
--- a/web/src/dropdown_widget.ts
+++ b/web/src/dropdown_widget.ts
@@ -106,6 +106,9 @@ export class DropdownWidget {
     disable_for_spectators: boolean;
     dropdown_input_visible_selector: string;
     prefer_top_start_placement: boolean;
+
+    // TODO: This is only used in one widget, with no implementation
+    // here, so should be generalized or reworked.
     item_clicked = false;
 
     constructor(options: DropdownWidgetOptions) {
```